Source/ProjectRimFactory/Storage/Building_MassStorageUnit.cs

```diff
@@ -1,55 +1,70 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Linq;
+﻿using System.Collections.Generic;
 using System.Text;
-using Verse;
-using RimWorld;
+using ProjectRimFactory.Common;
+using ProjectRimFactory.Common.HarmonyPatches;
 using ProjectRimFactory.Storage.Editables;
-using UnityEngine;
 using ProjectRimFactory.Storage.UI;
-using System.Collections;
-using ProjectRimFactory.Common.HarmonyPatches;
+using RimWorld;
+using UnityEngine;
+using Verse;
 
 namespace ProjectRimFactory.Storage
 {
     [StaticConstructorOnStartup]
-    public abstract class Building_MassStorageUnit : Building_Storage, IHideItem, IHideRightClickMenu, IForbidPawnOutputItem, IForbidPawnInputItem
+    public abstract class Building_MassStorageUnit : Building_Storage, IHideItem, IHideRightClickMenu,
+        IForbidPawnOutputItem, IForbidPawnInputItem
     {
-        static readonly Texture2D RenameTex = ContentFinder<Texture2D>.Get("UI/Buttons/Rename", true);
+        private static readonly Texture2D RenameTex = ContentFinder<Texture2D>.Get("UI/Buttons/Rename");

```

Comment:
Is this okay, or does it need to get filled in the StaticConstructorOnStartup?

Reply:
ContentFinder<T>.Get needs a static context for it to work otherwise you get an error about it running in a separate thread.